app/src/main/java/org/stepic/droid/notifications/NotificationTimeCheckerImpl.kt

```diff
@@ -31,8 +31,9 @@ class NotificationTimeCheckerImpl
     }
 
     override fun isNight(nowMillis: Long): Boolean {
-        val now = DateTime(nowMillis)
-        val nowHourInt = now.hourOfDay().get()
+        val calendar = Calendar.getInstance()
+        calendar.timeInMillis = nowMillis
+        val nowHourInt = calendar.get(Calendar.HOUR_OF_DAY)
         val result: Boolean = nowHourInt in startHour..(endHour - 1)
```

Comment:
There is `until` in Kotlin for such ranges
https://kotlinlang.org/docs/reference/ranges.html